### DISCUSSION TITLE
Structure backend

### DISCUSSION THREAD

**Discussion by Nocccer** [Heroic-Games-Launcher/HeroicGamesLauncher#486] [Ideas]
**Describe the solution you'd like**
Atm the backend is not well structured. All files are in the same folder. With more implementation it is hard to see what is common or launcher related. Would love to see some structure like in the frontend.

**Describe alternatives you've considered**
Example structure. Can be discussed.

Possible solutions:
* electron
    * common
        * util.ts
        * logger.ts
        * ...
    * legendary
        * ...
    * gog
        * ...
    * main.ts

**Comment by flavioislima** [comment]
Lets convert this into a discussion since this is not an issue.

**Comment by ghost** [comment]
I don't think we need a refactor. The current approach is maintainable if we split the stuff into files and folders.
